demo/b2share_demo/cli.py

```diff
@@ -147,3 +149,28 @@ def import_v1_data(verbose, download, token,
     for d in dirlist:
         process_v1_record(d, indexer, base_url, logfile, verbose)
     logfile.close()
+    
+@demo.command()
+@with_appcontext
+@click.argument('base_url')
+def generate_pid_migrator(base_url):
+    url = base_url + "api/records"
+    params = {}
+    params['size'] = 1000
+    params['page'] = 1
+    response = requests.get(url,params)
+    recs = json.loads(response.text)['hits']['hits']
+    for rec in recs:
+        url_value = rec['links']['self'].replace("/api/records","/records")
+        if 'alternate_identifiers' in rec['metadata'].keys():
+            alt_ids = rec['metadata']['alternate_identifiers']
+            epic_url = None
```

Comment:
The url where the PID can be changed is not identical to the global handle URL. We must take the suffix part (UUID) and place it at the end of our own handle server (see `config['CFG_EPIC_BASEURL']`)